sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/BaseCrossPlatformDriver.java

```diff
@@ -72,4 +75,14 @@ public void login() {
         new WebDriverWait(driver, 30).until(ExpectedConditions.
                 elementToBeClickable(MobileBy.AccessibilityId("login"))).click();
     }
+
+    public void verticalSwipe(String locator) throws InterruptedException {
```

Comment:
For this the application needs a fix, will push updated app and test in the next PR